### DISCUSSION TITLE
Is Solidus 2.11.x version having security support?

### DISCUSSION THREAD

**Discussion by roymj** [solidusio/solidus#4458] [Troubleshooting] [answered]
Hi Team

Have a query - As per the documentation https://solidus.io/security/, it is mentioned 2.11.x is not having security support after 18 months (EOL - 2022-04-23). 

However from the github solidus release, 2.11.17 was released 4 days back. 

Any confirmation that 2.11.x will continue to have support?

**Comment by jarednorman** [comment, marked as answer]
We have been being liberal with backporting patches. With Solidus v3.2 being released very soon, I would not count on it continuing to receive security patches. It's important for all stores to stay up to date with Solidus. The more you put it off, the harder it gets, and eventually you're stuck on versions of Solidus, Rails, and other dependencies that aren't getting security patches.
